controller/sentry/error.go

```diff
@@ -0,0 +1,9 @@
+package sentry
```

Comment:
I'd like to move `controller/sentry/error.go` -> `controller/internal/sentry/error.go` to not make it a part of the API.